products/revenue_analytics/backend/hogql_queries/revenue_analytics_overview_query_runner.py

```diff
@@ -15,7 +20,56 @@ class RevenueAnalyticsOverviewQueryRunner(RevenueAnalyticsQueryRunner):
     cached_response: CachedRevenueAnalyticsOverviewQueryResponse
 
     def to_query(self) -> ast.SelectQuery:
```

Comment:
@robbie-c Is it worth it to define these AST queries in a constant and simply return them rather than generating them at runtime all the time? Is it that much faster? I'm asking you because you investigated a little, don't worry if you don't have the answer

I know you were planning on a cache or something, but since that isn't here yet, this is something we could to improve latency at the cost of some memory